packages/orchestration/src/service.js

```diff
@@ -59,7 +59,7 @@ export const Proto3Shape = {
 };
 
 export const ChainAccountI = M.interface('ChainAccount', {
-  getAccountAddress: M.call().returns(M.string()),
+  getAddress: M.call().returns(M.string()),
```

Comment:
This should also return a ChainAddress.

Reply:
otherwise it cannot be passed as an arg to the various "send" operations

Reply:
[440fc82](https://github.com/Agoric/agoric-sdk/pull/9312/commits/440fc82285db7a5580f4261c7155863b140393ec)